app/controllers/api/v1/events_controller.rb

```diff
@@ -4,29 +4,14 @@ def show
   end
 
   def create
-    @event = Event.new(event)
-    if @event.save
-      render status: :ok
-    else
-      render json: {
-        message: 'Validation Failed',
-        errors: @event.errors.full_messages
-      }, status: :unprocessable_entity
-    end
+    @event = Event.create!(event)
```

Comment:
this and the update action (next comment) fail without the ! 
Do you want me to look for a way to do it without it?